python/travis/prepare_python_tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+#
+#  Copyright 2015-2017 TWO SIGMA OPEN SOURCE, LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+set -v
+
+cd ..
+make dist
+cd python
+
+curl http://d3kbcqa49mib13.cloudfront.net/spark-2.1.1-bin-hadoop2.7.tgz -o spark.tgz
```

Comment:
@leifwalsh good point. I've updated the Travis-URL to use the `apache.cs.utah.edu` mirror instead - assuming it is much more stable than the seemingly random Cloudfront URL.